README.md

```diff
@@ -0,0 +1,47 @@
+# BitlBee
+
+![](http://bitlbee.org/style/logo.png)
+
+[![Build Status](https://travis-ci.org/bitlbee/bitlbee.svg)](https://travis-ci.org/bitlbee/bitlbee)
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/4028/badge.svg)](https://scan.coverity.com/projects/4028)
+
+An IRC to other chat networks gateway
+
+Main website: http://www.bitlbee.org/
+
+Bug tracker: http://bugs.bitlbee.org/
+
+Wiki: http://wiki.bitlbee.org/
+
+License: GPLv2
+
+## Development
+
+Use github pull requests to submit patches.
+
+Mappings of bzr revisions to git commits: [html][], [raw][] (created using [git-bzr-rev-map][])
```

Comment:
Good point. Can it be just the "raw" one? (the one that's a tab separated text file). They are kinda big and the fancy one is a bit redundant.

```
232K    raw.txt
736K    fancy.html
```

And filename `doc/git-bzr-rev-map`?